components/molecule/selectPopover/src/index.js

```diff
@@ -79,6 +80,11 @@ function MoleculeSelectPopover({
     onCustomAction()
   }
 
+  const handleOnClickCancelButton = () => {
```

Comment:
For convention should be `handleClickCancelButton` without the `on` if is a handler function